docs/credhub-security-faq.md

```diff
@@ -51,13 +51,29 @@ TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
 ## How are Keys internally managed?
 
 If CredHub's third-party integration with a Luna SafeNet HSM is
-enabled, then CredHub has no knowledge of key values. If CredHub is
-started in internal encryption mode, CredHub deterministically
-generates a site-specific AES256 key on startup. CredHub concatenates
-a user-defined password from its configuration file with a
-randomly-generated salt stored in its database and hashes the
-resulting string to deterministically generate an AES256 key, which it
-holds in memory for the lifetime of the server.
+enabled, then CredHub has no knowledge of key values.
+
+If CredHub is started in internal encryption mode:
+
+* If a new encryption key is provided, CredHub will generate a new
```

Comment:
Should this say "If a new encryption password is provided"? Is it a password or a key? Same for the bullet below.